client-cli/pom.xml

```diff
@@ -46,6 +46,11 @@
             <groupId>org.bouncycastle</groupId>
             <artifactId>bcprov-jdk15on</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.nimbusds</groupId>
+            <artifactId>nimbus-jose-jwt</artifactId>
+            <version>8.2.1</version>
```

Comment:
Do we care to define the version in the properties section and reference the property here?

Reply:
The only time the property is really needed is when we have multiple dependencies of the same group with different artifact IDs but the same version. If we use Nimbus JWT anywhere else we'll just remove the version here and declare it in pom's dependencyManagement section.